packages/common/core/package.json

```diff
@@ -4,7 +4,7 @@
     "type": "commonjs",
     "dependencies": {
         "ts-morph": "17.0.1",
-        "tslib": "^2.5.0",
+        "tslib": "^2.6.1",
```

Comment:
As mentioned this was originaly required to support an NX issue of not correctly identifying the lib. Do the plugins work as expected if we remove this dependency altogether from this file?

Reply:
Unfortunately not, without specifying this in the common-core then all plugins have tslib set to a fixed version as their peer dependency.  (e.g `"tslib": "2.6.2"`) . E2E tests in this repo will still pass, however, client projects who are not on the same tslib version will be unable to run `npm i`